lib/Selenium/Remote/Driver/ActionChains.pm

```diff
@@ -0,0 +1,155 @@
+package Selenium::Remote::Driver::ActionChains; 
+# ABSTRACT: Action chains for Selenium::Remote::Driver
+use Moo; 
+
+has 'driver' => ( 
+    is => 'rw', 
+);
+
+has 'actions' => ( 
+    is => 'lazy', 
+    builder => sub { [] },
+    clearer => 1,
+);
+
+sub perform { 
+    my $self = shift; 
+    foreach my $action (@{$self->actions}) { 
+        $action->();
+    }
```

Comment:
actually yeah related to https://github.com/gempesaw/Selenium-Remote-Driver/pull/204/files#r30234682 if we are re-using the same action plans object, do we need to clear `$self->actions` after `perform`ing it? otherwise the next time we call perform we'll get all the old actions. 

or am i missing something :)